lib/helpers/copyDynamicImportChunks.js

```diff
@@ -0,0 +1,23 @@
+const { join } = require("path");
+const { copySync, readdirSync } = require("fs-extra");
+const { logTitle } = require("../helpers/logger");
+const { NEXT_DIST_DIR } = require("../config");
+
+// Check if there are dynamic import chunks and copy to the necessary function dir
+const copyDynamicImportChunks = (functionPath) => {
```

Comment:
Does this mean we're copying all `chunks` to all functions?

I was also worried users can control the `chunks` configuration until I found [this](https://github.com/vercel/next.js/issues/8834)

Reply:
yeah, it does. i think it should be okay for now and can definitely revisit later if issues arise from this implementation. reading the lambdas for the chunk names should be an "if absolutely necessary" approach imo 😂